src/main/java/com/google/api/codegen/metacode/InitCodeNode.java

```diff
@@ -353,7 +356,7 @@ private static void validateKeyValue(TypeModel parentType, String key) {
     if (parentType.isMap()) {
       TypeModel keyType = parentType.getMapKeyField().getType();
       validateValue(keyType, key);
-    } else if (parentType.isRepeated()) {
+    } else if (parentType.isRepeated() && parentType.getApiSource().equals(ApiSource.PROTO)) {
```

Comment:
Same question as above

Reply:
Removed for now.
This may become a problem again when unit tests are generated from Discovery documents.